com.unity.render-pipelines.universal/Editor/ShaderPreprocessor.cs

```diff
@@ -109,11 +109,7 @@ bool StripUnusedPass(ShaderFeatures features, ShaderSnippetData snippetData)
 
         bool StripUnusedFeatures(ShaderFeatures features, Shader shader, ShaderSnippetData snippetData, ShaderCompilerData compilerData)
         {
-#if URP_ENABLE_DEBUG_DISPLAY
             bool stripDebugDisplayShaders = !Debug.isDebugBuild;
```

Comment:
We need a follow up PR that introduces option in URP to force strip / include debug in build.

Reply:
Maybe something for when global settings land?

Reply:
Yes, that's my plan - to move the "debug shaders enabled" option from HDRP asset into HDRP global asset, and at the same time, introduce the same option in URP global asset. 

The question is, is this okay until then? I don't believe I can wait for URP global settings to land and still make the change before cutoff. Do you think this type of change can happen after cutoff?